retrofit-converters/simplexml/src/test/java/retrofit/converter/SimpleXMLConverterTest.java

```diff
@@ -9,65 +15,55 @@
 import org.simpleframework.xml.stream.Format;
 import org.simpleframework.xml.stream.HyphenStyle;
 import org.simpleframework.xml.stream.Verbosity;
-import retrofit.mime.TypedByteArray;
-import retrofit.mime.TypedInput;
-import retrofit.mime.TypedOutput;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 public class SimpleXMLConverterTest {
-	private static final String MIME_TYPE = "application/xml; charset=UTF-8";
+	private static final MediaType MEDIA_TYPE = MediaType.parse("application/xml; charset=UTF-8");
```

Comment:
Yeah wow. Not sure what's happening here. Both of these files were contributed and maybe they contain tabs? Will reformat.